content/js/index.md

```diff
@@ -12,6 +12,7 @@ application at the Skygear [Developer Portal](https://portal.skygear.io)
 website. After you registered, go to the **INFO** tab and copy down your
 `Server EndPoint` and `API Key`.
 
+![Example Image](/assets/common/portal-endpoint-apikey.png)
```

Comment:
- A more accurate description other than "Example Image"?
- The image has to be clickable to enlarge it